src/Altinn.Platform/Altinn.Platform.Profile/UnitTests/UserProfileTests.cs

```diff
@@ -0,0 +1,67 @@
+using Altinn.App.IntegrationTests;
+using Altinn.Platform.Profile.Models;
+using Altinn.Platform.Profile.UnitTests.Utils;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace UnitTests
+{
+    public class UserProfileTests : IClassFixture<CustomWebApplicationFactory<Altinn.Platform.Profile.Startup>>
```

Comment:
We could also use WebApplicationFactory<TStartup> directly. Doesn't look like CustomWebApplicationFactory adds anything yet.